### ISSUE TITLE
Batch files for block embeddings and other clean-up

### ISSUE THREAD

**Issue by schrum2** [schrum2/MarioDiffusion#151]
After closing the block/tile-embedding issue, I merged my dev_schrum branch into your dev_fletcher branch, so you have the up-to-date version of my block-embedding training script. I want you to do a bit of clean-up.

Get rid of files for training on just SMB1 data,

Make sure the unconditional training script has similar naming conventions and embedding length flexibility like the Mar1and2-conditional-embeddings.bat file, which I have made some changes to.

Delete and remove the "_alt" training script for block2vec.

After all of this, do quick checks that unconditional and conditional batch files work for training diffusion models with embedding vectors of length 10 (as I have already confirmed). These should be quick sanity checks, so you can terminate the training as soon as the generated samples show evidence of looking ok (no need to go all the way to 500 epochs).

After this, keep reducing the vector length to push the limit of what is possible. See how the results look with embedding vectors of length 8, 6, and 4, and post results here (focus on conditional models for these tests, and allow them to run as long as needed until results look good or training completes).

**Comment by fwilson12** [comment]
The recent commit in #96 removes the files, tagged the wrong issue number in the commit summary

**Comment by fwilson12** [comment]
Ran both the conditional and unconditional models and confirmed that their samples looked good after 60 epochs. Now moving on to training conditional models with block2vec models with fewer embedding dims

**Comment by fwilson12** [comment]
After confirming the conditional model trained on the 8-dimensional block2vec embeddings produced good results (at least for 100 epochs):

<img width="256" height="256" alt="Image" src="https://github.com/user-attachments/assets/ab5a7a94-49ba-4d57-8ce8-65a272b98fc0" />
<img width="256" height="256" alt="Image" src="https://github.com/user-attachments/assets/e15d802b-771e-46e0-bb5c-29051a39a115" />
<img width="256" height="256" alt="Image" src="https://github.com/user-attachments/assets/a1cec4b9-cb5b-4c11-87a1-642f9ba43657" />

I trained another conditional model on a 6-dimensional embedding model (all the way to 500 epochs this time), and again, the results were satisfactory:

<img width="256" height="256" alt="Image" src="https://github.com/user-attachments/assets/13490178-7a26-476e-acff-f43243abd8c6" />
<img width="256" height="256" alt="Image" src="https://github.com/user-attachments/assets/bbdb15f7-3287-4b61-9ffa-fe6c11f0fa5b" />
<img width="256" height="256" alt="Image" src="https://github.com/user-attachments/assets/88a4fa1f-9d52-4f92-8e79-aaf245d60b71" />


One thing to note is that the main pipe sections generated by the models consisted exclusively of the right-pipe block; this appears in the samples from the 6-dim model and in the limited samples from the 8-dim model, but may just be a one-off thing and would be rectified in independent training runs.

**Comment by fwilson12** [comment]
Going to close for now, but worth further testing if we want to use these smaller latent space models in the future
